docs/system-administration/DNS/debian-stretch.md

```diff
@@ -0,0 +1,723 @@
+---
+title: Debian 9.13 (Stretch) DNS Configuration.
+summary: A guide on how to setup Primary and Secondary caching servers with zone authority under Debian (stretch).
```

Comment:
I have been looking that. I think it can be simplified to a Bind9 guide. There does not appear to be any difference between stretch and Buster.

Reply:
I think they initial configs are a bit different between distros, but I don't have access to any other distros these days.

Reply:
It might be good to break this out in the future as DNS/BIND9/Debian, DNS/BIND9/"other Distro", DNS/Unbound/ DNS/PowerDNS.

Reply:
It's been a long time since I have looked at other distros, but I recall that some used 'named' instead of bind for directory names.

https://www.linuxtechi.com/setup-bind-server-centos-8-rhel-8/

In this guide "named" is used instead of "bind" or "bind9" so there is some difference.